FoodPin/Controller/RestaurantDetailViewController.swift

```diff
@@ -33,6 +30,32 @@ class RestaurantDetailViewController: UIViewController {
         
         navigationController?.hidesBarsOnSwipe = false
         navigationController?.navigationBar.barStyle = .black
+        
+        guard let restaurantName = restaurant?.name else {
+            return
+        }
+        headerView.nameLabel.text = restaurantName
```

Comment:
if you check in UIKIT, the text property of a UILabel is declared as String?. So you don't need to cast to String before assigning an optional to it.
And for future reference, a guard cast is useful for example when we need to use that variable multiple times, but for single uses we should try using the ternary operator.